SyncUp.Web/src/components/UploadImage.vue

```diff
@@ -6,7 +6,8 @@
       accept="image/*"
       style="display: none"
     />
-    <v-hover v-slot="{ isHovering, props }">
+    <v-skeleton-loader v-if="busy" type="image" width="300" />
```

Comment:
This is ugly and jumps the UI around, but the focus wasn't UI... I just wanted to have some visual indicator that the image was busy.

Reply:
There should have been an image placeholder, but the skeleton might be a better idea.